geomstats/visualization.py

```diff
@@ -21,6 +23,21 @@
                'H2_poincare_disk', 'H2_poincare_half_plane', 'H2_klein_disk',
                'poincare_polydisk']
 
+EPS = 1e-8
+
+
+def nice_matplotlib():
+    """Wrapper function for nice plotting style."""
+    plt.style.use('seaborn')
+    fontsize = 15
+    matplotlib.rc('font', size=fontsize)
+    matplotlib.rc('text', usetex=True)
+    matplotlib.rc('legend', fontsize=fontsize)
+    matplotlib.rc('font',
```

Comment:
Don't you get: INFO: font.family must be one of (serif, sans-serif, cursive, monospace) when text.usetex is True. serif will be used by default.?